scripts/configure-cluster/common/auth.sh

```diff
@@ -1,30 +1,10 @@
 #!/usr/bin/env bash
 
-set -e
-
-HTPASSWD_FILE="./htpass"
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+HTPASSWD_FILE="${SCRIPTPATH}/test-htpasswd"
 HTPASSWD_SECRET="htpasswd-secret"
```

Comment:
are these used now - I see a file for passwords already created

Reply:
Yes, they are used. It points to the pre-existing htpassword file.

Reply:
the `HTPASSWD_SECRET` is pointing to the file? because then this variable name is super confusing

Reply:
HTPASSWD_SECRET is the name of the secret that is created on the cluster using the `HTPASSWD_FILE`. We then create a htpasswd auth provider that is linked to this secret i.e told to use it.